yrs/src/block.rs

```diff
@@ -991,18 +998,29 @@ impl Item {
     }
 }
 
+impl PartialEq for Item {
+    fn eq(&self, other: &Self) -> bool {
+        self.try_eq(other)
+            .expect(format!("Item equality was not possible: {} vs {}", self, other).as_str())
+    }
+}
+
 #[derive(PartialEq, PartialOrd, Eq, Ord, Debug, Clone)]
 pub struct SplittableString {
     content: SmallString<[u8; 8]>,
-    utf16_len: usize,
```

Comment:
Since `Item::len` now contains precomputed field length, it doesn't have to reside here anymore.